lib/log.dart

```diff
@@ -30,3 +30,48 @@ bool debugLog(String message) {
   }());
   return true;
 }
+
+/// Display an error message in a [SnackBar].
+///
+/// This shows a [SnackBar] containing the message after [ZulipApp] becomes ready,
+/// otherwise logs it to the console.
```

Comment:
```suggestion
/// This shows a [SnackBar] containing the message if [ZulipApp] is ready,
/// otherwise logs it to the console.
```

(similarly with `reportErrorToUserInDialog` below)